0006-cmx2cdp-protocol/README.md

```diff
@@ -1,84 +1,92 @@
 ---
-name: CMX to CDP protocol
-shortname: 6/CMX2CDP
-status: raw
+name: Configuration Data Transport protocol
+shortname: 6/CDT
+status: draft
 editor: Andrew Pasika <apasika@cybervisiontech.com>
 ---
 
-## Introduction
+- [Introduction](#introduction)
+- [Language](#language)
+- [Requirements and constraints](#requirements-and-constraints)
+- [Design](#Design)
+  - [Configuration pull](#configuration-pull)
+    - [Subject structure](#subject-structure)
+    - [Targeted message types](#targeted-message-types)
+  - [Configuration push](#configuration-push)
+    - [Subject structure](#subject-structure)
 
-The CMX2CDP protocol is a [3/Messaging IPC][3/MIPC] protocol extension.
+## Introduction
 
-CMX2CDP is the protocol designed to communicate endpoint configurations from _configuration data provider_
-implementations to the _configuration management extensions_.
+Configuration Data Transport (CDT) protocol is designed to communicate endpoint configuration data between Kaa services.
 
-## Requirements and constraints
-### Problems and possible solutions
+In Kaa architecture, some services are designed to manage endpoint configuration in different ways — store it, process it, publish it, etc.
+One of the core features in such context is the ability to store and retrieve endpoint configurations.
 
-1. _Configuration push._
+For this purpose, the CDT protocol was designed.
```

Comment:
Since this is an RFC, I'd refrain from using past tense here. We want people to contribute on the RFCs, so they are technically still in the process of development.